internal/metamorph/store/postgresql/migrations/000016_multiple_callbacks.up.sql

```diff
@@ -0,0 +1,17 @@
+-- Step 1: Add the new 'callback' column
+ALTER TABLE metamorph.transactions
+    ADD COLUMN callbacks JSONB;
+
+-- Step 2: Populate the 'callback' column with data from 'callback_url' and 'callback_token'
+UPDATE metamorph.transactions
+SET callbacks = json_build_array(
+    json_build_object(
+        'callback_url', callback_url,
+        'callback_token', callback_token
+    )
+);
```

Comment:
nit: In order not to have this security warning: https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&pullRequest=539&types=BUG&severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&inNewCodePeriod=true&sinceLeakPeriod=true&id=bitcoin-sv_arc

add
```
WHERE 'callback_url' != '' OR 'callback_token' != ''
```

Reply:
I used `IS NOT NULL` because `!= ''` cause another error